libgtpv2c/src/decoder.c

```diff
@@ -834,7 +834,35 @@ decode_bearer_context_modified_ie_t(uint8_t *buf,
 
 	return count;
 }
+/* 8.13 Protocol Configuration Options (PCO) 
+ * 10.5.6.3 Protocol configuration options in 3GPP TS 24.008 */
 
+static int 
```

Comment:
Rarely changed code and rarely anyone is going to look into this code. I have put the spec reference which will help in knowing.. Variable names seems to be good enough to know that its extension bit and configuration protocol.

I will add more comments..but I wanted to avoid lots of macros.